src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/mapView.js

```diff
@@ -111,7 +119,7 @@ class MapViewVisualization extends Visualization {
         let controller = ok(parsedData.controller) ? parsedData.controller : true;
 
         const deckgl = new deck.DeckGL({
-            container: 'map',
+            container: id,
             mapboxApiAccessToken: TOKEN,
             mapStyle,
             initialViewState: {longitude,latitude,zoom,pitch},
```

Comment:
This is very bad. It even can cause our GUI to crash and needs to be fixed ASAP. We should never reinitialize visualizations on new incomming data!